UI/src/app/text-input/text-input.component.ts

```diff
@@ -7,10 +7,12 @@ import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 })
 export class TextInputComponent implements OnInit {
 
-    textInput: string;
+    textInput: string = 'Plot a pie chart for number of employees in each department';
```

Comment:
Why do you assign that string twice?

Reply:
no no..  This needs to changed back to textInput: string;